### PR TITLE
Generic CRUD Service was added

### PR DESCRIPTION
**Pull request by Its-OP**
Abstract base CRUD service was added

### CODE REVIEW

**Review by ihor-il — COMMENTED** [review]
I'd like to see DTOs as params and not the entities. In that way we can keep mapper inside service and not request it in controllers

**Comment by Its-OP** [comment]
> 

Can't agree with it. Services belong to the business layer of application, while DTOs are clearly parts of the presentation layer. They are something that user interacts with.

File: backend/Tasque.BLL/Services/EntityCrudService.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+
+        public TEntity Create(TEntity entity)
+        {
+            _db.Set<TEntity>().Add(entity);
+            _db.SaveChanges();
+            return entity;
+        }
+
+        public TEntity? ReadOne(int id)
```
Comment (ismv) [inline backend/Tasque.BLL/Services/EntityCrudService.cs]: I know that R in CRUD stands for Read but conventionally we use Get for these kinds of methods
GetById, GetAll, etc

Comment (Its-OP) [reply backend/Tasque.BLL/Services/EntityCrudService.cs]: Renamed in both service and controller

**Pull request closed without merging**